feature/experimental/policy/otg_tests/prefix_set_test/prefix_set_test.go

```diff
@@ -17,23 +17,31 @@ package prefix_set_test
 import (
 	"context"
 	"testing"
+	"time"
 
 	"github.com/openconfig/featureprofiles/internal/fptest"
-	"github.com/openconfig/featureprofiles/internal/helpers"
+	spb "github.com/openconfig/gnoi/system"
 	"github.com/openconfig/ondatra"
 	"github.com/openconfig/ondatra/gnmi"
 	"github.com/openconfig/ondatra/gnmi/oc"
 )
 
 const (
 	prefixSetA = "PFX_SET_A"
+	tag3IPv4   = "TAG_3_IPV4"
 	pfx1       = "10.240.31.48/28"
 	pfx2       = "173.36.128.0/20"
 	pfx3       = "173.36.144.0/20"
 	pfx4       = "10.240.31.64/28"
 	mskLen     = "exact"
 )
 
+var (
+	processName = map[ondatra.Vendor]string{
+		ondatra.ARISTA: "Octa",
+	}
+)
+
```

Comment:
This should go into /internal.  I've seen we've needed to reference processes and a findProcessByName function in  other featureprofiles tests as well.  (one example https://github.com/openconfig/featureprofiles/blob/e0876594d87d88658675cfa075229e29e50fe640/feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go#L29-L34)

Probably a new folder such as /internal/system

This variable could be named routingProcessName (to differentiate from other processes that could be queried).

Reply:
will create a separate CL to take of this refatoring